### ISSUE TITLE
Bug with WPML and Use directory for default language

### ISSUE THREAD

**Issue by Ercogx** [samiahmedsiddiqui/custom-permalinks#36]
If I use WPML and have checked for "Use directory for default language" and I have the same links for the two languages, I have problem with Language switcher - it makes a link view "site.com/en/posts" "site.com//en/posts". It happens in function custom_permalinks_trailingslash 
```
if ( trim( $_CPRegisteredURL, '/' ) == trim( $request, '/' ) ) {
      if ( isset( $url['path'] ) ) {
        return ( $string{0} == '/' ? '/' : '' ) . trailingslashit( $url['path'] ) . $_CPRegisteredURL;
      } else {
        return ( $string{0} == '/' ? '/' : '' ) . $_CPRegisteredURL;
      }
    }
```

**Comment by samiahmedsiddiqui** [comment]
Hi @Ercogx 

I think, this issue is already fixed. Can you please try this [Alpha version](https://github.com/yasglobal/custom-permalinks/releases/tag/v1.6.0-alpha).

If you have any development server then it will be really easy otherwise I recommend you to install this alpha version for the **testing purpose** only on production.

**Comment by Ercogx** [comment]
Cool it works? but unfortunately not for all cases. If on one page I use a normal slag not custom permalink and on another I use a custom permalink with the same link I will get the same bug)

**Comment by samiahmedsiddiqui** [comment]
@Ercogx Do you have any development server?

**Comment by Ercogx** [comment]
So far, the site is only local, but I was able to reproduce the error on the test site

[http://wp.little-about-web.top/en/test/](http://wp.little-about-web.top/en/test/)

**Comment by samiahmedsiddiqui** [comment]
@Ercogx Please provide me the access on [this](mailto:sami.siddiqui@yasglobal.com) email address. Also, please sent me the test link so I can quickly test the things.

**Comment by Ercogx** [comment]
@samiahmedsiddiqui sent email

**Comment by samiahmedsiddiqui** [comment]
@Ercogx Issue fixed, please update to the [beta](https://github.com/samiahmedsiddiqui/custom-permalinks/releases/tag/v1.6.0-beta) version and test the issue again.

Let me know if it will not fixed for you.

**Comment by samiahmedsiddiqui** [comment]
@Ercogx Thank you so much for your support and testing the issues with me. I believe all your reported issues are sort out. I have officially released `v1.6.0` that contains all your fixes. 

Please feel free to open a new issue if you find any new issue.

Regards,
Sami
